src/Halogen/Portal.purs

```diff
@@ -21,20 +23,26 @@ import Halogen.VDom.Driver as VDom
 import Web.HTML (HTMLElement)
 import Type.Row as Row
 
-type InputFields query input output
+type InputFields n query input output
```

Comment:
I can certainly do this as a followup commit, but I prefer to take the monad parameter last:

```suggestion
type InputFields query input output n
```

to match with the order of type variables used in the `child` field below (and its typical place in type signatures)